introduction-to-datasets.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+When using the CLI, you define your dataset using the [`create`][create] command. When using the API, you define your dataset using [`DatasetDescriptor.Builder`][dataset-desc-bldr].
```

Comment:
The context of this line changed when I moved it here and added the note about the CLI. It had been in the section on properties. I've changed the link to the `Datasets.create` command, which will lead to `DatasetDescriptor`, etc.